github/repos_pages.go

```diff
@@ -75,6 +75,34 @@ func (s *RepositoriesService) EnablePages(ctx context.Context, owner, repo strin
 	return enable, resp, nil
 }
 
+// PagesUpdate sets up parameters needed to update a GitHub Pages site.
+type PagesUpdate struct {
+	CNAME  *string `json:"cname,omitempty"`
```

Comment:
Can you please add a comment above `CNAME` like this:

```
// CNAME represents a custom domain for the repository.
// Leaving CNAME empty will remove the custom domain.
```

Reply:
Sorry, wasn't expecting such a prompt response, was still poking at it.

Thoughts on the secondary type I've since added so this can remain optional without removing the custom domain?